packages/atomic-template/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@coveo/create-atomic",
```

Comment:
**For the next steps, I have a few questions!**

All the other projects use a separate generator package, or a plugin for the generators offered by the community (e.g. `create-react-app`)

I guess we will have to do the same for Atomic, I decided to name that package `@coveo/create-atomic` so that it works standalone when doing `npm init @coveo/atomic`. The CLI will instantiate a new project and set the .env variables.

My question is do you have a technology to recommend for a generator, I'm seeing the use of [@schematics/angular](https://angular.io/guide/schematics) in the angular project which appears to offers everything necessary but seems to be more for well, Angular. React & Vue also have their own tools.

Exploring and playing around I found a few possible options:
- [Yeoman Generator](https://yeoman.io/authoring/)
- [Plop](https://plopjs.com/)
- Not using any technology (or smaller separate packages) seems also possible, but perhaps counter-productive

The Atomic project that will be create will basically contain:
- index.html file with an Atomic interface
- a few CSS files
- the `search-token-server` leveraging the .env variables

PS: In the future, when we want to create a more complex setup that can be used to create custom Atomic component (see [design doc](https://docs.google.com/document/d/1QzqezewjxUNdd2MhEQ4bmMT_0K1PcQbF3FBhzR2yka4/edit#heading=h.fkb38024b4a9)) we would then have something similar to what Stencil offers with their `npm init stencil` command (see [create-stencil](https://www.npmjs.com/package/create-stencil))

Reply:
I think both Plop and Yeoman are good candidates for the @coveo/atomic project.

Reply:
I used Plop before, it does the job and I don't recall facing any bugs.